BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift

```diff
@@ -108,9 +108,7 @@ public class CollapsibleCardView: ShadowCardView, UIGestureRecognizerDelegate {
 
     lazy var titleLabel: UILabel = .build { label in
         label.adjustsFontForContentSizeCategory = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
-                                                            size: 15.0,
-                                                            weight: .semibold)
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
```

Comment:
Since this has a weight of semibold, lets use the bold styles: `FXFontStyles.Bold.subheadline.scaledFont()`